.github/workflows/release.yml

```diff
@@ -30,3 +30,38 @@ jobs:
             target/*.tar.gz
             target/rpm/jcliff/RPMS/noarch/*.rpm
           body: "Release ${{ steps.get_version.outputs.TAG_VERSION }}"
+      - name: Homebrew Release
+        env:
+          HOMEBREW_TAP_REPO: ${{ secrets.HOMEBREW_REPOSITORY }}
+          BREW_USER: ${{ secrets.BREW_USER }}
+          BREW_TOKEN: ${{ secrets.BREW_TOKEN }}
+        run: |
```

Comment:
@sabre1041 anyway to externalise this bash script into a separate file? It would be cleaner IMH.

Reply:
Sure. Just need to know where in the repository it should go

Reply:
I generally put this kind of scripts into src/build/bash.